persistent-test/src/CompositeTest.hs

```diff
@@ -109,7 +97,7 @@ specsWith runDb = describe "composite" $
       kp2 <- insert p2
       xs <- selectList [TestParentId <-. [kp1,kp2]] []
       length xs @== 2
-      let [e1@(Entity newkp1 newp1),e2@(Entity newkp2 newp2)] = xs
+      let [(Entity newkp1 newp1),(Entity newkp2 newp2)] = xs
```

Comment:
```suggestion
      [(Entity newkp1 newp1),(Entity newkp2 newp2)] <- pure xs
```

This uses hte `MonadFail` machinery instead of a partial pattern match.

Reply:
Sure, fine by me. I just removed the `e1` and `e2` because they weren't used anywhere P: